config/prow/cluster/monitoring/mixins/prometheus/prow_alerts.libsonnet

```diff
@@ -0,0 +1,23 @@
+{
+  prometheusAlerts+:: {
+    local componentName = $._config.components.monitoring,
+    groups+: [
+      {
+        name: 'prow',
+        rules: [
+          {
+            alert: 'prow-pod-crashlooping',
+            expr: 'rate(kube_pod_container_status_restarts_total{namespace=~"default|prow-monitoring",job="kube-state-metrics"}[5m]) * 60 * 5 > 0',
```

Comment:
FYI for the future: Prometheus has a convenient `increase(<metric>[<duration>])` function that can be used instead of `rate(<metric>[<duration>]) * <duration in seconds>`. It handles breaks in monotonicity just like `rate()`.